src/SprykerEco/Zed/Payone/Business/Payment/MethodMapper/CashOnDelivery.php

```diff
@@ -0,0 +1,228 @@
+        $refundContainer->setBankbranchcode($paymentEntity->getSpyPaymentPayoneDetail()->getBankBranchCode());
+        $refundContainer->setBankcheckdigit($paymentEntity->getSpyPaymentPayoneDetail()->getBankCheckDigit());
+        $refundContainer->setIban($paymentEntity->getSpyPaymentPayoneDetail()->getIban());
+        $refundContainer->setBic($paymentEntity->getSpyPaymentPayoneDetail()->getBic());
+
+        return $refundContainer;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PayoneBankAccountCheckTransfer $bankAccountCheckTransfer
+     *
+     * @return \SprykerEco\Zed\Payone\Business\Api\Request\Container\BankAccountCheckContainer
+     */
+    public function mapBankAccountCheck(PayoneBankAccountCheckTransfer $bankAccountCheckTransfer): BankAccountCheckContainer
+    {
+        $bankAccountCheckContainer = new BankAccountCheckContainer();
+
+        $bankAccountCheckContainer->setAid($this->getStandardParameter()->getAid());
+        $bankAccountCheckContainer->setBankCountry($bankAccountCheckTransfer->getBankCountry());
+        $bankAccountCheckContainer->setBankAccount($bankAccountCheckTransfer->getBankAccount());
+        $bankAccountCheckContainer->setBankCode($bankAccountCheckTransfer->getBankCode());
+        $bankAccountCheckContainer->setIban($bankAccountCheckTransfer->getIban());
+        $bankAccountCheckContainer->setBic($bankAccountCheckTransfer->getBic());
+        $bankAccountCheckContainer->setLanguage($this->getStandardParameter()->getLanguage());
+
+        return $bankAccountCheckContainer;
+    }
+
+    /**
+     * @param \Orm\Zed\Payone\Persistence\SpyPaymentPayone $paymentEntity
+     * @param \SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\AbstractAuthorizationContainer $authorizationContainer
+     *
+     * @return \SprykerEco\Zed\Payone\Business\Api\Request\Container\Authorization\AbstractAuthorizationContainer
+     */
+    protected function mapPaymentToAbstractAuthorization(
+        SpyPaymentPayone $paymentEntity,
+        AbstractAuthorizationContainer $authorizationContainer
+    ): AbstractAuthorizationContainer {
+        $paymentDetailEntity = $paymentEntity->getSpyPaymentPayoneDetail();
+
+        $authorizationContainer->setAid($this->getStandardParameter()->getAid());
+        $authorizationContainer->setClearingType(PayoneApiConstants::CLEARING_TYPE_CASH_ON_DELIVERY);
+        $authorizationContainer->setReference($paymentEntity->getReference());
+        $authorizationContainer->setAmount($paymentDetailEntity->getAmount());
+        $authorizationContainer->setCurrency($this->getStandardParameter()->getCurrency());
+        $authorizationContainer->setPaymentMethod($this->createPaymentMethodContainerFromPayment($paymentEntity));
+
+        $personalContainer = new PersonalContainer();
+        $this->mapBillingAddressToPersonalContainer($personalContainer, $paymentEntity);
```

Comment:
mapBillingAddressToPersonalContainer returns void?
In any case, it looks that you can move this to the separate method

Reply:
It`s very old code...moved partially to `buildPersonalContainer()`